src/System.Numerics.Vectors/ref/System.Numerics.Vectors.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <ProjectGuid>{650277B5-9423-4ACE-BB54-2659995B21C7}</ProjectGuid>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx' OR '$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
+	<DefineConstants Condition="'$(TargetGroup)'=='netcoreapp'">$(DefineConstants);netcoreapp</DefineConstants>
```

Comment:
Does this need to be `'$(TargetGroup)'=='netcoreapp' OR '$(TargetGroup)'=='uap' OR '$(TargetGroup)'=='uapaot'`?  We want this new API for UAP as well, right?

Reply:
Oops. I wanted to use `IsTargetingNetCoreApp` and did something different. Let me correct it.

Reply:
Note that the ref does not have uapaot-specific build, so you can omit that in the condition for ref.

Reply:
ok! skipping `uapaot` in that condition...